src/meetingsession/DefaultMeetingSession.ts

```diff
@@ -191,38 +191,37 @@ export default class DefaultMeetingSession implements MeetingSession, Destroyabl
       );
     }
 
-    if (this._configuration.enableUnifiedPlanForChromiumBasedBrowsers) {
-      if (browserBehavior.hasChromiumWebRTC()) {
-        this.logger.info('WebRTC unified plan for Chromium-based browsers is enabled');
-      } else {
-        this.logger.info(`WebRTC unified plan is required for ${browserBehavior.name()}`);
+    if (browserBehavior.requiresUnifiedPlan()) {
+      if (
+        browserBehavior.hasChromiumWebRTC() &&
+        !this._configuration.enableUnifiedPlanForChromiumBasedBrowsers
+      ) {
+        this.logger.error(
+          'Plan B SDP has been deprecated on Chromium-based browsers. ' +
+            `Overriding configured value of 'enableUnifiedPlanForChromiumBasedBrowsers' to be true`
+        );
       }
+      this.logger.info('WebRTC Unified Plan is enabled');
+    } else {
+      this.logger.info(`WebRTC Unified Plan is disabled on ${browser}`);
     }
 
     // Validation if a custom video uplink policy is specified
     if (this._configuration.videoUplinkBandwidthPolicy) {
       if (this.isSimulcastUplinkPolicy(this._configuration.videoUplinkBandwidthPolicy)) {
-        if (!this._configuration.enableUnifiedPlanForChromiumBasedBrowsers) {
+        if (!browserBehavior.hasChromiumWebRTC()) {
           throw new Error(
-            'Simulcast requires enabling WebRTC Unified Plan for Chromium-based browsers'
+            'Simulcast is only supported on Chromium-based browsers and enabled Unified Plan'
```

Comment:
I don't think we need to mention Unified Plan here since we no longer support plan-B for Chrome.

Reply:
Addressed